tools/component-pages-data.js

```diff
@@ -464,6 +464,27 @@ export const componentPagesData = {
 				'internal': false
 			},
 			'content': '\r\nA polymer-based widget that displays a list of collapsible components. When a collapsible component is clicked, it expands or collapses the associated content.\r\n\r\n<img width="845" alt="Accordion component" src="/img/screenshots/accordion.png">\r\n\r\nComponents:\r\n- `d2l-labs-accordion`\r\n- `d2l-labs-accordion-collapse`'
+		},
+		{
+			'frontMatter': {
+				'layout': 'layouts/component-issue',
+				'title': 'Navigation',
+				'fileName': 'd2l-navigation',
+				'repo': 'https://github.com/BrightspaceUI/navigation/',
```

Comment:
```suggestion
				'repo': 'https://github.com/BrightspaceUILabs/navigation',
```
I expect GitHub is just handling the redirect for us, but probably better to use the actual path?